### PR TITLE
DROID-1776 Gallery experience | Enhancement | Resolving deep link

### PR DESCRIPTION
**Pull request by konstantiniiv**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockRemote.kt [hunk 1]
```diff
@@ -412,4 +413,7 @@ interface BlockRemote {
     suspend fun stopSharingSpace(space: SpaceId)
 
     suspend fun getSpaceInviteLink(spaceId: SpaceId): SpaceInviteLink
+
+    suspend fun downloadGalleryManifest(command: Command.DownloadGalleryManifest): ManifestInfo?
+    suspend fun objectImportExperience(command: Command.ObjectImportExperience): Payload
```
Comment (uburoiubu) [inline data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockRemote.kt:418]: `importExperience`

File: domain/src/main/java/com/anytypeio/anytype/domain/block/repo/BlockRepository.kt [hunk 2]
```diff
@@ -455,4 +456,7 @@ interface BlockRepository {
     suspend fun stopSharingSpace(space: SpaceId)
 
     suspend fun getSpaceInviteLink(spaceId: SpaceId): SpaceInviteLink
+
+    suspend fun downloadGalleryManifest(command: Command.DownloadGalleryManifest): ManifestInfo?
+    suspend fun objectImportExperience(command: Command.ObjectImportExperience): Payload
```
Comment (uburoiubu) [inline domain/src/main/java/com/anytypeio/anytype/domain/block/repo/BlockRepository.kt:461]: `importExperience`

File: gallery-experience/src/main/java/com/anytypeio/anytype/gallery_experience/models/GalleryInstallationState.kt [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
     object Main : GalleryInstallationNavigation("main")
     object Spaces : GalleryInstallationNavigation("spaces")
     object Success : GalleryInstallationNavigation("success")
     object Error : GalleryInstallationNavigation("error")
     object Dismiss : GalleryInstallationNavigation("")
-}
\ No newline at end of file
+}
+
+data class SpaceView(
```
Comment (uburoiubu) [inline gallery-experience/src/main/java/com/anytypeio/anytype/gallery_experience/models/GalleryInstallationState.kt:27]: `GallerySpaceView`

**Review by uburoiubu — APPROVED** [review]

File: domain/src/main/java/com/anytypeio/anytype/domain/block/repo/BlockRepository.kt (hunk 2 above)
Comment (konstantiniiv) [reply domain/src/main/java/com/anytypeio/anytype/domain/block/repo/BlockRepository.kt]: done

File: data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockRemote.kt (hunk 1 above)
Comment (konstantiniiv) [reply data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockRemote.kt]: done

**Pull request merged**